docs/docusaurus/docs/cloud/alerts/alerts_lp.md

```diff
@@ -17,4 +17,5 @@ import OverviewCard from '@site/src/components/OverviewCard';
 <LinkCardGrid>
   <LinkCard topIcon label="Manage email alerts" description="Configure and manage email alerts for your Data Assets." to="/cloud/alerts/manage_email_alerts" icon="/img/email_action_icon.svg" />
   <LinkCard topIcon label="Trigger actions" description="Create and manage Actions based on the results of Validation runs." to="/cloud/alerts/trigger_actions" icon="/img/actions_icon.svg" />
+  <LinkCard topIcon label="Create a custom action" description="Create a custom Action." to="/cloud/alerts/custom_actions" icon="/img/actions_icon.svg" />
```

Comment:
The card description repeats the card title verbatim. Ideally these two pieces of text should be different. Here is a suggestion that gets at the value provided by custom actions

The new card has the same icon as an existing card on this page. Ideally, each card should have a distinct icon. Here's a suggestion for a different icon
<img width="58" height="63" alt="Screenshot 2025-11-20 at 1 36 15 PM" src="https://github.com/user-attachments/assets/d9b855ab-1094-467d-aa4b-a3a83f2485c2" />

```suggestion
  <LinkCard topIcon label="Create a custom action" description="Apply custom business logic based on Validation Results." to="/cloud/alerts/custom_actions" icon="/img/add_feature_icon.svg" />
```